INTMDB-284: Updated docs in alert configuration resource and datasource (#645)
mongodb/terraform-provider-mongodbatlas: website/docs/d/alert_configuration.html.markdown

* `datadog_region` - Region that indicates which API URL to use. Accepted regions are: `US`, `EU`. The default Datadog region is US.
* `delay_min` - Number of minutes to wait after an alert condition is detected before sending out the first notification.
* `email_address` - Email address to which alert notifications are sent. Required for the EMAIL notifications type.

> * `email_enabled` - Flag indicating email notifications must be sent. Atlas returns this value if you set `type_name` to `ORG`, `GROUP`, or `USER`.

Reviewer: ```suggestion
* `email_enabled` - Flag indicating email notifications should be sent. Atlas returns this value if `type_name` is set  to `ORG`, `GROUP`, or `USER`.
```

* `ops_genie_region` - Region that indicates which API URL to use. Accepted regions are: `US` ,`EU`. The default Opsgenie region is US.
* `org_name` - Flowdock organization name in lower-case letters. This is the name that appears after www.flowdock.com/app/ in the URL string. Required for the FLOWDOCK notifications type.
* `service_key` - PagerDuty service key. Required for the PAGER_DUTY notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.

> * `sms_enabled` - Flag indicating text notifications must be sent. Atlas returns this value if you set `type_name` to `ORG`, `GROUP`, or `USER`.

Reviewer: ```suggestion
* `sms_enabled` - Flag indicating text notifications should be sent. Atlas returns this value if `type_name` is set to `ORG`, `GROUP`, or `USER`.
```
